stdlib/sql/open.go

```diff
@@ -38,6 +38,8 @@ func getOpenFunc(driverName, dataSourceName string) openFunc {
 	switch driverName {
 	case "mssql", "sqlserver":
 		return mssqlOpenFunction(driverName, dataSourceName)
+	case "snowflake":
+		return snowflakeOpenFunction(driverName, dataSourceName)
```

Comment:
How does `driverName` get here? Is it possible that a user opens a connection to snowflake with `driverName != snowflake`? (Would let them bypass `DisableFileTranfers`)

Reply:
I think this is the only way to access it: https://github.com/snowflakedb/gosnowflake/blob/v1.6.7/driver.go#L54

That indicates the library registers itself with the sql library as the "snowflake" driver. I don't think there's another name it can be accessed by. I'm not the most familiar though and it all makes me a bit nervous.

If we're making a fork anyway, maybe we should just consider changing the code itself to statically always return false instead of having it as an option?

Reply:
fwiw, there's a lot of code in both `sql.from` and `sql.to` that relies on a fixed list of incoming driver name values. For example: https://github.com/influxdata/flux/blob/00ef526ad50e3fb2e7e9bee7dbcccae9ca22b34b/stdlib/sql/from.go#L116-L137

This specific block for looking up the opener could be made more explicit such that it errors when we see an unexpected name, regardless of how the `DisableFileTransfers` config is handled.